docs/runtimes/functions.md

```diff
@@ -58,6 +58,92 @@ handle the orchestration and monitoring of the distributed task.
 
 <img src="../_static/images/runtime-scaling.png" alt="runtime-scaling" width="400"/>
 
+## Applying storage configurations to functions
+
+In the vast majority of cases, an MLRun function would require access to storage. This storage
+may be used to provide inputs to the function such as data-sets to process or data-streams that contain input events.
+Storage would usually also be used to store function outputs and result artifacts, such as trained models or processed
+data-sets.
+
+As MLRun function may be distributed and executed in Kubernetes pods, the storage used would in most cases be a shared
```

Comment:
```suggestion
Sine MLRun functions may be distributed and executed in Kubernetes pods, the storage used would typically be shared, and execution pods would need some added configuration options applied to them so that the function code is able to access the designated storage.
```